.editorconfig

```diff
@@ -2,7 +2,7 @@ root = true
 
 [*]
 indent_style = tab
-indent_size = 1
+indent_size = 4
```

Comment:
This obviously assumes that this is a desirable change. I can modify it / remove it if not.